src/app/backend/handler/apihandler.go

```diff
@@ -321,11 +321,18 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 		apiV1Ws.GET("/horizontalpodautoscaler/{namespace}").
 			To(apiHandler.handleGetHorizontalPodAutoscalerList).
 			Writes(horizontalpodautoscaler.HorizontalPodAutoscalerList{}))
+	apiV1Ws.Route(
+		apiV1Ws.GET("/horizontalpodautoscaler/{kind}/{namespace}/{name}").
+			To(apiHandler.handleGetHorizontalPodAutoscalerListForResource).
+			Writes(horizontalpodautoscaler.HorizontalPodAutoscalerList{}))
 	apiV1Ws.Route(
 		apiV1Ws.GET("/horizontalpodautoscaler/{namespace}/{horizontalpodautoscaler}").
 			To(apiHandler.handleGetHorizontalPodAutoscalerDetail).
 			Writes(horizontalpodautoscaler.HorizontalPodAutoscalerDetail{}))
-
+	apiV1Ws.Route(
```

Comment:
It's the same as above?

Reply:
yes, it's the same, I have removed it. Thank you~ And I will help to add the front-end part later if nobody take it.